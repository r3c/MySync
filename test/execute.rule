compatibility 3600 1 echo -n > {}
name: echo -n > {}
    interval=2
    keep=3
