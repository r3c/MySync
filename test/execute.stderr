^compatibility: new backup file saved as '[^']*/compatibility\.[0-9]*'$
^09_AZ-az.rule: new backup file saved as '[^']*/09_AZ-az.rule\.[0-9]*'$
^warning: too-large: backup file '[^']*/too-large.[0-9]*' is larger than expected \(2 bytes\)$
^too-large: new backup file saved as '[^']*/too-large.[0-9]*'$
^warning: too-small: backup file '[^']*/too-small.[0-9]*' is smaller than expected \(1 bytes\)$
^too-small: new backup file saved as '[^']*/too-small.[0-9]*'$
^$
